portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,9 +40,16 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    //greets.add("Whats up");
-    //greets.add("Hi Dese!");
-    //greets.add("Hows it going");
+    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Did you miss to commit these before this pull request? This file came up in the last PR as well